src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5320,8 +5320,7 @@ public override IOperation VisitEventAssignment(IEventAssignmentOperation operat
             else
             {
                 Debug.Assert(operation.EventReference != null);
-                Debug.Assert(operation.EventReference.Kind == OperationKind.Invalid);
```

Comment:
> Debug.Assert(operation.EventReference.Kind == OperationKind.Invalid); [](start = 15, length = 70)

Could you please add a test for a scenario that would cause this assert to fail?

Reply:
@AlekseyTs added.